packages/babeld-auto-gw-mode/files/etc/watchping/wan-ok.d/babeld-gw

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/lua
+--! Copyright (C) 2021 Santiago Piccinini <spiccinini@altermundi.net>
+--!
+--! This is free software, licensed under the GNU Affero General Public License v3.
+
+local ubus = require "ubus"
+
+local conn = ubus.connect()
+if not conn then
+    error("Failed to connect to ubus")
+end
+
+local wan4_status = conn:call("network.interface.wan", "status", {})
+
+for _, route4 in ipairs(wan4_status.route) do
+    if route4.target == "0.0.0.0" and route4.mask == 0 then
+        --TODO: maybe add also src config?
```

Comment:
As this route will not really used to route actual packets, but just for babeld to read it and spread to the network there is no advantage in specifying the source address

Reply:
Thanks for the clarification, I will remove the comment